### PR TITLE
Fix undefined array key breadcrumb

### PR DESCRIPTION
**Pull request by ausi**

### CODE REVIEW

File: core-bundle/contao/dca/tl_templates.php [hunk 1]
```diff
@@ -218,6 +218,7 @@ public function addBreadcrumb()
 		$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['root'] = array($strNode);
 
 		// Insert breadcrumb menu
+		$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['breadcrumb'] ??= '';
 		$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['breadcrumb'] .= '
```
Comment (ausi) [inline core-bundle/contao/dca/tl_templates.php]: Is this the correct fix, or should the next line just be `=` instead of `.=`?

**Comment by ausi** [comment]
Failing CI unrelated, see #5094

File: core-bundle/contao/dca/tl_templates.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/contao/dca/tl_templates.php]: The latter, I think. Theoretically, there could be a callback that has already added something to `$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['breadcrumb']` but I don‘t think that is a supported use-case, is it?

Comment (ausi) [inline core-bundle/contao/dca/tl_templates.php]:
```suggestion
		$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['breadcrumb'] = '
```

Comment (ausi) [reply core-bundle/contao/dca/tl_templates.php]: Done in ad05006a10846545d9eb2ca3d8c1ba3584c133ce

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**